DevCycle.SDK.Server.Local/Api/DVCLocalClient.cs

```diff
@@ -233,6 +233,8 @@ public void FlushEvents()
         
         public override void Dispose()
         {
+            Task flushTask = eventQueue.FlushEvents();
```

Comment:
does triggering this also prevent new events from being tracked at all? the other sdks have that protection built in